app/src/main/res/values/strings.xml

```diff
@@ -1,51 +1,62 @@
+    <string name="lorem_ipsum">
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam in scelerisque sem. Mauris
+    volutpat, dolor id interdum ullamcorper, risus dolor egestas lectus, sit amet mattis purus
+    dui nec risus. Maecenas non sodales nisi, vel dictum dolor. Class aptent taciti sociosqu ad
+    litora torquent per conubia nostra, per inceptos himenaeos. Suspendisse blandit eleifend
+    diam, vel rutrum tellus vulputate quis. Aliquam eget libero aliquet, imperdiet nisl a,
+    ornare ex. Sed rhoncus est ut libero porta lobortis. Fusce in dictum tellus.\n\n
+    Suspendisse interdum ornare ante. Aliquam nec cursus lorem. Morbi id magna felis. Vivamus
+    egestas, est a condimentum egestas, turpis nisl iaculis ipsum, in dictum tellus dolor sed
+    neque. Morbi tellus erat, dapibus ut sem a, iaculis tincidunt dui. Interdum et malesuada
+    fames ac ante ipsum primis in faucibus. Curabitur et eros porttitor, ultricies urna vitae,
+    molestie nibh. Phasellus at commodo eros, non aliquet metus. Sed maximus nisl nec dolor
+    bibendum, vel congue leo egestas.
+</string>
+    <string name="next">Next</string>
+    <string name="notification_message">SimplyWords is currently running in the background.</string>
+    <string name="notification_title">SimplyWords is running</string>
+    <string name="or_lowercased">or</string>
     <string name="original_text">Original text</string>
-    <string name="simplified_text">Simplified text</string>
-    <string name="content_description_home">Homepage of SimplyWords</string>
+    <string name="permission_overlay_message">We need a special permission to display over other apps. We\'ll use this to display the simplified text over the original text. If you do not grant this permission, you might not be able to make full use of this app.</string>
+    <string name="permission_camera_message">We need access to your camera for us to scan and simplify the words. </string>
     <string name="permission_title">Permission required</string>
-    <string name="permission_message">We need a special permission to display over other apps. We\'ll use this to display the simplified text over the original text. If you do not grant this permission, you might not be able to make full use of this app.</string>
-    <string name="notification_title">SimplyWords is running</string>
-    <string name="notification_message">SimplyWords is currently running in the background.</string>
+    <string name="previous">Previous</string>
+    <string name="profile">Profile</string>
     <string name="recents">Recents</string>
-    <string name="simplify_description">View your past queries and configure our Simplifier tool here.</string>
+    <string name="recentActivityLabel">Recent Activity</string>
+    <string name="settings">Settings</string>
     <string name="settings_description">Change various settings for the app here.</string>
+    <string name="simplify_camera">Take a picture.</string>
```

Comment:
Since this is used in a button, maybe we can get rid of the full stop at the end so it fits in with everything else?

```suggestion
    <string name="simplify_camera">Take a picture</string>
```

Reply:
Temporary fix to the issue, declared 
`    <uses-permission android:name="android.permission.QUERY_ALL_PACKAGES" />`
    in AndroidManifest